src/pages/category/categorylist.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const CategoryPage = () => (
+  <div>Category List</div>
+);
```

Comment:
- [ ] Instead of displaying the text `Category List`, this component should be displaying a `Check Status` button. Please work on this.

  Here is the requirement for that:
  > - Should display a button "Check status" only.
  
  ## Screenshot:
  
  ![Screenshot 2022-07-16 124721](https://user-images.githubusercontent.com/45002654/179350218-44021a7c-31ef-480b-bfc7-f0ed3ef6f3a5.png)